new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/InstallParams.java

```diff
@@ -3,27 +3,100 @@
 
 package oracle.weblogic.kubernetes.actions.impl.primitive;
 
-// Presents all parameters that the test InstallParams needs.
+/**
+ * Presents all parameters for downloading a tool.
+ */
 
 public class InstallParams {
+  public static final String WIT_TYPE = "WIT";
+  public static final String WDT_TYPE = "WDT";
 
-  private String type;
+  public static final String DEFAULT_WIT_DOWNLOAD_URL = "https://github.com//oracle/weblogic-image-tool";
+  public static final String DEFAULT_WIT_VERSION    = "release-1.8.3";
+  public static final String WIT_FILE_NAME  = "imagetool.zip";
+
+  public static final String DEFAULT_WDT_DOWNLOAD_URL = "https://github.com//oracle/weblogic-deploy-tooling";
+  public static final String DEFAULT_WDT_VERSION    = "weblogic-deploy-tooling-1.7.2";
+  public static final String WDT_FILE_NAME  = "weblogic-deploy.zip";
+
+  // WIT or WDT
+  private String type = "WIT";
+  // The version of the tool
   private String version;
+  // The download site location
   private String location;
+  // Whether verify before download
+  private boolean verify = true;
+  // Whether the download zip file needs to be unziped
+  private boolean unzip = false;
+  // Whether the output of the command is redirected to system out
+  private boolean redirect = true;
 
   public InstallParams type(String type) {
     this.type = type;
     return this;
   }
 
+  public String getType() {
```

Comment:
fluent api's do not use the prefix `get`

```
public String thing() {
  return thing;
}

public Whatever thing(String newThing) {
  this.thing = newThing;
  return this;
}
```

Reply:
will do

Reply:
done